spotifycli/test/test_spotifycli.py

```diff
@@ -0,0 +1,163 @@
+from io import StringIO
+from pathlib import Path
+from subprocess import PIPE
+from unittest import skip
+from unittest.mock import patch
+
+import dbus
+import dbusmock
+
+from spotifycli.spotifycli import main
+from spotifycli.version import __version__
+
+
+class TestSpotifyCLI(dbusmock.DBusTestCase):
```

Comment:
Here we are creating a new unit test class that inherits from the dbusmock test case for convenient access to things like `spawn_server`.  Otherwise it's really just a plain `unittest.TestCase`